README.md

```diff
@@ -1,127 +1,98 @@
 
-## How Can I Help?
+### Governance
 
-+ Try [TiDB](https://github.com/pingcap/tidb)! If you like it, star, share, and recommend it. 
+TiDB has the following types of groups that are officially supported:
 
-+ To dig deeper, read a design doc, e.g. [architecture](https://www.pingcap.com/blog/2017-05-23-perconalive17/), [How we build TiDB](https://www.pingcap.com/blog/2016-10-17-how-we-build-tidb/)
+* **Special Interest Groups (SIGs)** are persistent open groups that focus on a
+  part of the TiDB. SIGs must have open and transparent proceedings. Anyone is
+  welcome to participate and contribute provided they follow the TiDB [Code of
+  Conduct](./CODE_OF_CONDUCT.md). The purpose of a SIG is to own and develop a
+  part of TiDB.
 
-+ Start with improving the documentation. You can search [open issues](https://github.com/pingcap/docs/issues) to find inspiration for what to work on.
+* **Working Groups (WGs)** are temporary groups that are formed to address
+  issues that cross SIG boundaries. Working groups do not own any code or other
+  long term artifacts. Working groups can report back and act through involved
+  SIGs.
 
-+ There's always code that can be clarified and variables or functions that can be renamed or commented.
+### How to contribute
 
-+ There's always a need for more test coverage. You can start with an area that has [low coverage](https://codecov.io/gh/pingcap/tidb).
+Contributions are welcomed and greatly appreciated. See
+[CONTRIBUTING.md](./CONTRIBUTING.md)
+for details on submitting patches and the contribution workflow.
 
-+ Give a talk at a local meetup or conference.
+You can start by finding an existing issue with the
+[help-wanted](https://github.com/pingcap/tidb/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+label in the tidb repo. These issues are well suited for new contributors. BTW,
+there's always a need for more test coverage. You can also start with an area
+that has [low coverage](https://codecov.io/gh/pingcap/tidb).
 
-## Learn to Use TiDB
+You can also join one of the [active working groups](./working-groups) to
+participate in a community activity. Start attending working group meetings and
+join the slack channel.
 
-Links in [https://pingcap.com/docs](https://pingcap.com/docs) lead to many relevant topics, including:
+Some useful documents:
 
-+ TiDB Introduction
+* [Github Workflow](./contributors/workflow.md)
+* [SRE-BOT Command Help](./contributors/command-help.md)
+* [Become a Committer](./become-a-committer.md)
 
-+ TiDB Quick Start Guide
+## Learning Resources
 
-+ TiDB Operations Guide
+* [How we build TiDB](https://www.pingcap.com/blog/2016-10-17-how-we-build-tidb/)
+* [A Brief Introduction of TiDB](https://www.pingcap.com/blog/2017-05-23-perconalive17/)
+* [https://pingcap.com/docs](https://pingcap.com/docs)
+* [TiDB 源码阅读系列文章](https://pingcap.com/blog-cn/#TiDB-%E6%BA%90%E7%A0%81%E9%98%85%E8%AF%BB), in chinese
```

Comment:
Suggest removing the Chinese links. Maybe some other articles in English, like TiDB Internal I II III? ...

Reply:
Have we translated the TiDB Internal articles?

Reply:
We have. 
https://pingcap.com/blog/2017-07-11-tidbinternal1/
https://pingcap.com/blog/2017-07-11-tidbinternal2/
https://pingcap.com/blog/2017-07-11-tidbinternal3/

Reply:
They're focused on the tidb architecture, while these Chinese articles are focused on the source code reading, they're different for the developers. I think both of them should be added to the learning resources.

Reply:
BTW, we're going to use a big section to collect all the learning resources. It will be refactored in the future.

Reply:
OK